src/main/java/com/jcabi/dynamo/Region.java

```diff
@@ -49,6 +49,10 @@
  *
  * <pre> Region region = new ReRegion(new Region.Simple(credentials));</pre>
  *
+ * <p>After all operations with the region are finished, it can be optionally
+ * shutdown invoking {@link AmazonDynamoDB#shutdown()} in order to explicitly
+ * release any open resources.
```

Comment:
optionally? what will happen if I don't do a shutdown? I think we should explain a bit more here...